Documentation/General/emergentArchitectureDesign/emergentArchitectureDesign.tex

```diff
@@ -0,0 +1,180 @@
+\chapter{Introduction}
+\setheader{Introduction}
+This document provides a sketch of the system that is going to be built during the context project multimedia services. The architecture of the system is explained in the form of high level components of the system. These components are split in to sub components and sub-systems.
+
+\section{Design goals}
+The following design goals will be maintained throughout the project:
+\begin{itemize}
+
+\item
+\textbf{Deployability}
+\\
+The system will be developed in such a way that we can always deploy the most current version (Continuous Integration \cite{Duvall}).
+Being able to deploy the system at any time allows us to keep the work required for a release manegable.
+Because the current version should be deployable at anytime, developers are enforced to only alter the system in a non-breaking way.
+Another advantage is that smaller releases\footnote{With smaller we mean releases that touches only a few lines or a few files of the codebase.} imply a lower possibility of introducing bugs.
```

Comment:
"With smaller we mean releases that touches only a few lines or a few files of the codebase."
->
"We define \"smaller releases\" as a relatively small change to the codebase."